dashboard/error_handler.py

```diff
@@ -0,0 +1,30 @@
+from collections import namedtuple
```

Comment:
Add encoding to the top of the file `` # -*- coding: utf-8 -*- ``. Also insert a new line between imports.